extensions/wikia/BannerNotifications/js/BannerNotifications.js

```diff
@@ -211,19 +209,28 @@
 	 */
 	function handleScrolling() {
 		var containerTop,
-			notificationWrapper = $pageContainer.children('.banner-notifications-wrapper');
+			notificationWrapper = $pageContainer.children('.banner-notifications-wrapper'),
+			headerBottom;
 
 		if (!$pageContainer.length || !notificationWrapper.length) {
 			return;
 		}
 
 		// get the position of the wrapper element relative to the top of the viewport
-		containerTop = $pageContainer[0].getBoundingClientRect().top;
+		containerTop = $pageContainer.get(0).getBoundingClientRect().top;
+		headerBottom = $header.get(0).getBoundingClientRect().bottom;
 
-		if (containerTop < headerHeight) {
-			notificationWrapper.addClass('float').css('top', headerHeight);
+		if (containerTop < headerBottom) {
+			if (!notificationWrapper.hasClass('float')) {
```

Comment:
It's not that easy. If I move it above then `else` block will be way more complicated.